src/queries.js

```diff
@@ -60,23 +71,20 @@ function editorActivityQuery (editorbucket, limit){
     'editorbucket = "' + editorbucket + '";',
     'events = flood(query_bucket(editorbucket));',
     'files = sort_by_duration(merge_events_by_keys(events, ["file", "language"]));',
-    'files = limit_events(files, ' + limit + ');',
+    `files = limit_events(files, ${limit});`,
     'languages = sort_by_duration(merge_events_by_keys(events, ["language"]));',
-    'languages = limit_events(languages, ' + limit + ');',
+    `languages = limit_events(languages, ${limit});`,
     'projects = sort_by_duration(merge_events_by_keys(events, ["project"]));',
-    'projects = limit_events(projects, ' + limit + ');',
+    `projects = limit_events(projects, ${limit});`,
     'duration = sum_durations(events);',
     'RETURN = {"files": files, "languages": languages, "projects": projects, "duration": duration};'
   ];
 }
 
 function dailyActivityQuery (afkbucket){
-  return [
-    'afkbucket = "' + afkbucket + '";',
-    'not_afk = flood(query_bucket(afkbucket));',
-    'not_afk = merge_events_by_keys(not_afk, ["status"]);',
-    'RETURN = not_afk;'
-  ];
+  return events_active().concat([
```

Comment:
This will make a huge performance hit, the aw-watcher-web bucket is much larger than aw-watcher-afk.
Might still be fast enough still though, I have no idea so you have to test.

Reply:
Ah shit, true. Don't know what I was thinking.